modules/mlops/kubeflow-platform/README.md

```diff
@@ -0,0 +1,79 @@
+## Introduction
+This is an implmententation of [Kubeflow](https://www.kubeflow.org/docs/) via [Kubeflow-on-AWS](https://awslabs.github.io/kubeflow-manifests/docs/).
+It leverages the [Kubeflow-on-AWS github repo](https://github.com/awslabs/kubeflow-manifests) and the [Kubeflow-Manifests Github repo](https://github.com/kubeflow/manifests).
+
+
+
+
+## Description
+
+This is a seedfarmer / ADDF wrapped implementation based off Kubeflow-on-AWS.  This allows customization of the deployment via the module's `deplopyspec`.
+It currently supports only the following:
+- `vanilla` deployment via the `kustomize` deployment method
+- Kubeflow v1.6.1
+- AWSKubebuild 1.0.0
+
+*** NOTE: this module does not support earlier versions of Kubeflow v1.6.1 as specified in the branches of [Kubeflow-on-AWS](https://github.com/awslabs/kubeflow-manifests)
+
+Since this module is extenisble, additions and modifications are encouraged.
+
+
+
+## Prerequisites
```

Comment:
We assume (at least at this point) that EKS cluster is deployed by using the `core/eks` module.
It could be helpful to mention that Kubeflow will be deployed on the EKS cluster named
"addf-<deployment_name>-core-eks-cluster" under this assumption.

Reply:
Yes...using these modules implies that you are using the eks cluster ADDF modules create...but it is not a requirement.

Reply:
Right, not requirement. 

The only my concern is if users can easily identify the EKS cluster they are using for the Kubeflow deployment.

But, to respect the modularity of `mlops/kubeflow-platform` and `mlops/kubeflow-users`, it makes sense to not referring to the EKS cluster name coming from `core/eks` here.

Reply:
Look at some of the other modules.  This is the pattern we have followed, I would like to remain consistent.